src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01 Browser.md

```diff
@@ -3,22 +3,17 @@ title: "Browser"
-| [browser.contexts()](/javascript-api/xk6-browser/browser/contexts)                             | Allows you to access all open browser contexts.                                                                 |
-| [browser.isConnected](/javascript-api/xk6-browser/browser/isconnected)                         | Indicates whether the WebSocket connection to the browser application is active or not.                         |
-| 🚧 [browser.newBrowserCDPSession()](/javascript-api/xk6-browser/browser/newbrowsercdpsession) | Allows you to access Browser's CDP session and talk directly to the browser application using the CDP protocol. |
-| [browser.newContext([options])](/javascript-api/xk6-browser/browser/newcontext/)               | Creates and returns a new browser context.                                                                      |
-| [browser.newPage([options])](/javascript-api/xk6-browser/browser/newpage)                      | Creates a new Page in a new BrowserContext and returns the page.                                                |
-| 🚧 [browser.on('disconnected')](/javascript-api/xk6-browser/browser/on)                       | Detects events from the browser application.                                                                    |
-| ❌ [browser.startTracing()](/javascript-api/xk6-browser/browser/starttracing)                  | Starts Chromium Tracing for debugging a Page.                                                                   |
-| ❌ [browser.stopTracing()](/javascript-api/xk6-browser/browser/stoptracing)                    | Stops the tracing created by the browser.startTracing() method.                                                 |
-| [browser.version()](/javascript-api/xk6-browser/browser/version)                               | Returns the browser application's version.                                                                      |
+| Method                                                                                                                                                                         | Description                                                                                                                                           |
+|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| <nobr>[browser.close()](/javascript-api/xk6-browser/browser/close)</nobr>                                                                                                      | Closes the browser and all of its pages (if any were opened).                                                                                         |
```

Comment:
🤦 I was going to check whether `nobr` works across all browsers, but then forgot. I'll change it to work with `div` and `CSS`.

I'll try to merge in main into #582 and then merge #582 into this branch so that I can use the PixelPoint tooltop component.

Reply:
Resolved in: 10d88016f50f77c8f68354edf7b13aaaf7ee7e30